App/src/main/java/com/gluonhq/chat/views/HomePresenter.java

```diff
@@ -27,12 +32,22 @@ public void initialize() {
 
         homeView.showingProperty().addListener((obs, ov, nv) -> {
             if (nv) {
-                configView(tablet, orientation);
+                setupView(tablet, orientation);
+            }
+        });
+    }
+
+    private void setupView(boolean tablet, Orientation orientation) {
+        service.getName(n -> {
+            if (n != null && n.get() != null && !n.get().isEmpty()) {
+                setupAuthenticatedView(tablet, orientation);
+            } else {
+                AppViewManager.LOGIN_VIEW.switchView(ViewStackPolicy.SKIP);
```

Comment:
How do I go to the Login View when I am already in the Chat View(?) showing all the messages?

Reply:
That happens (on desktop, right?) probably because you already have a `userinfo` file in ~/.gluon from devoxxApp or alike... Just remove it

Reply:
Couldn't find any `userinfo` file in `~/.gluon`. However, I had `name` and `usernames` files, both of which were empty. Deleting these files did the trick.